rest-api/src/main/java/gov/cms/qpp/conversion/api/exceptions/InvalidFileTypeException.java

```diff
@@ -0,0 +1,15 @@
+package gov.cms.qpp.conversion.api.exceptions;
+
+/**
+ * Exception for handling an invalid file type
+ */
+public class InvalidFileTypeException extends RuntimeException {
+
+	/**
+	 * Constructor to call RuntimeException
+	 * @param message
+	 */
+	public InvalidFileTypeException(String message) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Document the parameter(s): message [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUndocumentedApi)